src/Gateway/PayplugGateway.php

```diff
@@ -684,6 +684,12 @@ public function filter_tokens($tokens, $user_id, $gateway_id)
     public function payment_fields()
     {
         $description = $this->get_description();
+
+		//add fields of IP to the description
+		if($this->payment_method === 'integrated'){
+			$description .= IntegratedPayment::template_form();
```

Comment:
no need to add it to the `$description` you can simple print `IntegratedPayment::template_form()`